### DISCUSSION TITLE
Ability to make panels collapsible and expandable

### DISCUSSION THREAD

**Discussion by smbera** [foxglove/community#621] [Q&A]
There is a situation where there are many panels, at this moment, I only focus on the "3D" panel, but the display space of the "3D" panel is very small. So I have to shrink other panels by dragging, so that the display space of the "3D" panel will be larger.

![image](https://user-images.githubusercontent.com/23401125/215306216-1325944d-2835-4c4f-9a0d-3f47ca138d32.png)

**Is there an ability to make panels collapsible and expandable?**
![image](https://user-images.githubusercontent.com/23401125/215306262-5a34c92d-e235-4c08-847b-35b8047d88c5.png)

**Comment by smbera** [comment]
Here is the PR for this question.
https://github.com/foxglove/studio/pull/5260

https://user-images.githubusercontent.com/23401125/215308646-ed099967-77c1-4396-aaaf-c8bf4a883fc6.mp4

**Comment by amacneil** [comment]
Have you tried the "Fullscreen" option available for each panel?

<img width="309" alt="image" src="https://user-images.githubusercontent.com/637671/215308128-aef75e93-b7f7-4498-8ce9-6e7d4a5ccd58.png">

**Reply by smbera** [reply]
Yes, already tried. If I focus on multiple panels, fullscreen doesn't fit multiple panels, and full screen will block the left sidebar.
